lib/src/upgrader.dart

```diff
@@ -44,6 +46,11 @@ class Upgrader {
   /// When an appcast is configured for iOS, the iTunes lookup is not used.
   AppcastConfiguration? appcastConfig;
 
+  /// An optional value that can override the default packageName when
+  /// attempting to reach the Google Play Store. This is useful if your app has
+  /// a different package name in the Play Store.
+  String? androidId;
```

Comment:
A better name for this variable would be `applicationId`, since that is what the Android docs use. See https://developer.android.com/studio/build/application-id